CHANGELOG.md

```diff
@@ -0,0 +1,30 @@
+# Changelog
+
+All notable changes specific to pantheon-upstreams/drupal-project are noted here.
+
+
+
+## Pantheon Update #1 - 2021-02-03
+
+### Added
+
+- Add changelog to track Pantheon-specific changes to pantheon-upstreams/pantheon-project. (#10)
+
+- Two repositories for release management. Pull requests accepted at https://github.com/pantheon-systems/drupal-project and releases located at https://github.com/pantheon-upstreams/drupal-project. (#4,#5)
+
+
+
+### Changed
+
+- Allow site-level customization of Drush version. (#5)
+
+- Allow sites to downgrade to Drupal 8.8.  (#6)
+- Use optimized autoloader in Test and Live, but not Dev or Multidev (#8)
```

Comment:
Looks like we have empty lines between each item, except for this line? Seems like we can get rid of the empty lines? The markdown renders fine though, so defer to you.